pkg/parcacol/querier.go

```diff
@@ -381,12 +388,6 @@ func (q *Querier) queryRangeDelta(ctx context.Context, filterExpr logicalplan.Ex
 		}
 	}
 
-	for name, index := range columnIndices {
```

Comment:
I think this check was useful. Maybe we pre-populate the indexes with `-1` to check if we have found an index for each of them. Otherwise we silently err on the side of using index 0.

Reply:
I had *int before but the -1 is nicer. Will update.